# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: aecd7795068a
# Full Hash: aecd7795068a42ee91f3ea624e52233a32341199
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2022-07-20 09:59:35
# Regressor Bug: 1773701
# File Overlap Count: 1
# Description:
#   Bug 1773701 - Part 3: Add email tracking telemetry probes. r=dimi
#   
#   This patch adds two telemetry probes.
#   1. The counter of classified third-party email trackers in different
#      categories.
# ==============================================================================

diff -r d393067941bd -r aecd7795068a modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Tue Jul 19 20:04:24 2022 +0000
+++ b/modules/libpref/init/all.js	Tue Jul 19 20:04:25 2022 +0000
@@ -3798,6 +3798,9 @@
 pref("urlclassifier.trackingAnnotationSkipURLs", "google.com/recaptcha/,*.google.com/recaptcha/");
 pref("privacy.rejectForeign.allowList", "");
 
+// The list of email webapp sites
+pref("privacy.trackingprotection.emailtracking.webapp.domains", "mail.163.com,mail.aol.com,fastmail.com,webmail.gandi.net,mail.google.com,navigator-bs.gmx.com,app.hey.com,horde.org/apps/webmail,hushmail.com,icloud.com/mail,kolabnow.com,laposte.net/accueil,mail.lycos.com,mail.com/mail/,mail.ru,mailfence.com,outlook.live.com,email-postaci.com/,posteo.de,mail.protonmail.com,app.rackspace.com,mail.rediff.com,emailmg.ipage.com,runbox.com,mail.sina.com.cn,tutanota.com,mail.yahoo.com,mail.yandex.com,mail.zimbra.com,zoho.com/mail/");
+
 // Number of random entries to send with a gethash request
 pref("urlclassifier.gethashnoise", 4);
 